cmd/balance-agent/main.go

```diff
@@ -303,6 +304,30 @@ func getApp() *cli.App {
 			Value:  "mainnet",
 			Hidden: true,
 		},
+		&cli.StringFlag{
+			Name:   "grpc-url",
```

Comment:
grpc-url is ambiguos. It can be url to node or backend api.

because we are data company my suggestion is: datastore-url